STEERING-COMMITTEE.md

```diff
@@ -46,6 +49,40 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
+
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+
+1. Review and approve definition of Knative Core scope.
+
+1. Only the Trademark Committee can redefine its own charter and authority over
+   Knative Trademark, conformance, and brand decisions.
+
+## Trademark Committee Composition
+
+The Trademark Committee is composed of company representatives from
+companies which have substantially contributed to Knative over the last
+year. For the 2020 year, one seat is assigned to each of Google, IBM/RedHat, and
+VMware. The committee shall review this composition at least annualy and update
```

Comment:
Which committee?

Reply:
"The Trademark Committee"

Reply:
It wouldn't be that hard to create eligibility criteria for new contributing organizations here, which might forestall arguements.  Like "50 or more PRs in the last year, including contributions to the test suite, and ownership of one or more products based on Knative".  Maybe there should be some kind of financial committement as well, but since the finances of the project are not public I'm not sure how you'd measure that.